packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text from '../Text';
+
+function Term({
+  label,
+  after,
+  endAlign,
+  keyTextRegular,
+  children,
+  cx,
+  styles,
+}: {
+  label: string;
+  after?: NonNullable<React.ReactNode>;
```

Comment:
`NonNullable` and an optional prop (`?`) are conflicting. let's make it optional and accepts are ReactNode.

`after?: React.ReactNode;`